plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/AbstractXmlTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.eclipse.test.util;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.w3c.dom.Document;
+
+public class AbstractXmlTest {
+
+  @Rule public PluginXmlDocument pluginXmlDocument = new PluginXmlDocument();
```

Comment:
Just a thought: I don't think `PluginXmlDocument` will be re-used elsewhere. Why not merge it into `AbstractXmlTest`? (I'm not strong about this.)